jdaviz/configs/default/plugins/export_plot/export_plot.py

```diff
@@ -74,3 +112,184 @@ def vue_save_figure(self, filetype):
+        if not isinstance(viewer, BqplotImageView):  # Profile viewer in glue-jupyter cannot do this
+            raise TypeError(f"Movie for {viewer.__class__.__name__} is not supported.")
+        if viewer.shape is None:
+            raise ValueError("Selected viewer has no display shape.")
+
+        if fps is None:
+            fps = float(self.movie_fps)
+        if fps <= 0:
+            raise ValueError("Invalid frame rate, must be positive non-zero value.")
+
+        if filename is None:
+            if self.movie_filename:
+                filename = self.movie_filename
+            else:
+                raise ValueError("Invalid filename.")
+
+        # Make sure file does not end up in weird places in standalone mode.
+        path = os.path.dirname(filename)
+        if path and not os.path.exists(path):
+            raise ValueError(f"Invalid path={path}")
+        elif (not path or path.startswith("..")) and os.environ.get("JDAVIZ_START_DIR", ""):  # noqa: E501 # pragma: no cover
+            filename = os.path.join(os.environ["JDAVIZ_START_DIR"], filename)
+
+        if i_start is None:
+            i_start = int(self.i_start)
+
+        if i_end is None:
+            i_end = int(self.i_end)
+
+        # No wrapping. Forward only.
+        slice_plg = self.app._jdaviz_helper.plugins["Slice"]._obj
+        if i_start < 0:  # pragma: no cover
+            i_start = 0
+        if i_end > slice_plg.max_value:  # pragma: no cover
+            i_end = slice_plg.max_value
+        if i_end <= i_start:
+            raise ValueError(f"No frames to write: i_start={i_start}, i_end={i_end}")
+
+        threading.Thread(
+            target=lambda: self._save_movie(i_start, i_end, fps, filename, rm_temp_files)
+        ).start()
+
+        return os.path.abspath(filename)
+
+    def vue_save_movie(self, filetype):  # pragma: no cover
+        """
+        Callback for save movie events in the front end viewer toolbars. Uses
+        the bqplot.Figure save methods.
+        """
+        try:
+            filename = self.save_movie(filetype=filetype)
+        except Exception as err:  # pragma: no cover
+            self.hub.broadcast(SnackbarMessage(
+                f"Error saving {self.movie_filename}: {err!r}", sender=self, color="error"))
+        else:
+            # Let the user know where we saved the file.
+            # NOTE: Because of threading, this will be emitted even as movie as recording.
+            self.hub.broadcast(SnackbarMessage(
+                f"Movie being saved to {filename} for slices {self.i_start} to {self.i_end}, "
+                f"inclusive, at {self.movie_fps} FPS.",
+                sender=self, color="success"))
+
+    def vue_kill_recording(self, *args):  # pragma: no cover
+        self.movie_kill = True
+        self.hub.broadcast(SnackbarMessage(
+            f"Movie recording interrupted by user, {self.movie_filename} will be deleted.",
+            sender=self, color="warning"))

```

Comment:
It might be nice to have this exposed in the public API as well (or to have `save_movie` be blocking and a keyboard interrupt to trigger this).  Although if public, `interrupt` or `stop` might be a bit... friendlier... than "kill".

Reply:
I don't feel like making it too easy to kill with API. That button is supposed to be a last resort button.

Reply:
But I will rename.